src/resources/js/components/AccountEdition.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="container">
+    <h2>My Account</h2>
+
+    <div v-if="showMessage" class="error">
+      <h3>{{ message }}</h3>
+    </div>
+
+    <form @submit.prevent="submitChange()">
+      <label for="UpdateFirstName">
+        <input v-model="updateUserInfo.fname" type="text" :placeholder="this.loggedUser.fname" />
```

Comment:
The old value should be placed in a `value` attribute.

`The placeholder attribute specifies a short hint that describes the expected value of an input field` 

[Source](https://www.w3schools.com/tags/att_input_placeholder.asp)

Reply:
I appears that the value attribut is already overridden by v-model, so i have remove the binding in placeholder.
the error output:```
  :value="this.loggedUser.fname" conflicts with v-model on the same element because the latter already expands to a value binding internally
```

Reply:
The you need to do `updateUserInfo = loggedUser` (Eache attribute ofc)